tests/helpers/dom.js

```diff
@@ -0,0 +1,31 @@
+/* global SimpleDOM */
+
+export function childNodesLength(element) {
+  let length = 0;
+  let node = element.firstChild;
+  while (node) {
+    length++;
+    node = node.nextSibling;
+  }
+  return length;
+}
+
+export function outerHTML(node) {
+  if (node.outerHTML) {
+    return node.outerHTML;
+  } else {
+    let serializer = new SimpleDOM.HTMLSerializer([]);
+    return serializer.serialize(node);
+  }
+}
+
+export function innerHTML(parentNode) {
```

Comment:
See: https://github.com/krisselden/simple-dom/pull/18
If we get simple-dom version bumped this can just use `serializer.serializeChildren(node)`